src/app/view/components/_common/iframe/iframe-component/iframe.component.ts

```diff
@@ -44,6 +41,12 @@ export class IframeComponent implements OnInit {
         this.iframeOverlayService.overlay.subscribe((val) => (this.showOverlay = val));
         this.element.nativeElement.style.height = this.getIframeHeight(window.innerHeight);
 
+        this.dotIframeService.reloaded().subscribe(() => {
+            if (this.getIframeWindow()) {
```

Comment:
why this?

Reply:
Honestly, not 100% sure, but if I don't do the check first is trow an ```location``` undefined error, not sure if it's something with the iframe, not sure.